src/components/common/NodensFooter.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <footer class="main-footer black white--text">
+    <div class="upper-footer py-12" style="background-color: #72aaf5">
+      <v-container>
+        <v-row>
+          <v-col col="12">
+            <h3 class="font-weight-bold mb-4 black--text display-2">
+              Want to know more?
+            </h3>
+            <p class="text-h4 black--text">
+              Schedule a meeting or send a message
+            </p>
+            <div class="mt-16">
+              <v-btn
```

Comment:
There was no need to make this button dynamic. Not all buttons will be used for routing. What if user wants button click to be handled differently. Please use static buttons for this, thanks.